frontend/src/app/constants/index.tsx

```diff
@@ -1,4 +1,4 @@
-export const COVID_RESULTS = {
```

Comment:
Not for this PR but I was wondering why some fields like symptoms are directly mapped into the numeric codes whereas these are our own values and mapped by the backend. Seems like we should be consistent. Like, have the backend send us the official values for the map or if they're relatively stable we could compile them on the frontend from what the hub needs to see.

Reply:
This is due to optimizing for expediency. The initial backend took what ever value the frontend sent it. When integrating with the data hub we needed to map the values we had to the HL7 values it expects. 

Long term I think we want to pull this information from the data hub but we need to wait until that is available